src/lib/Scenes/Artwork/Components/OtherWorks/OtherWorks.tsx

```diff
@@ -8,7 +8,7 @@ import { createFragmentContainer, graphql } from "react-relay"
 import { ContextGridCTA } from "./ContextGridCTA"
 import { Header } from "./Header"
 
-export const populatedGrids = (grids: OtherWorks_artwork["contextGrids"]) => {
+export const populatedGrids = grids => {
```

Comment:
Why remove the type here? Functions preferably do type their params.

Reply:
I removed this typing since it no longer applies (it did when we were re-using the fragment container's query).

But I can add something back.